Compilation issue

```
ERROR in src/app/components/user-list/user-list.component.html:2:18 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Argument of type 'UserAccount[] | Promise<UserAccount[]>' is not assignable to parameter of type 'Promise<UserAccount[]> | null | undefined'.
      Type 'UserAccount[]' is missing the following properties from type 'Promise<UserAccount[]>': then, catch, [Symbol.toStringTag], finally

2     <div *ngIf="(members$ | async) as members; else loading">
                   ~~~~~~~~~

  src/app/components/user-list/user-list.component.ts:6:16
    6   templateUrl: './user-list.component.html',
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component UserListComponent.
```

--- a/src/app/components/user-list/user-list.component.html
+++ b/src/app/components/user-list/user-list.component.html
@@ -1,10 +1,5 @@
 <div class="container rounded">
-    <div *ngIf="(members$ | async) as members; else loading">
-        <div *ngFor="let member of members">
-            <img [src]="member.profilePictureUrl"><span><a [href]="'/user/' + member.id">{{member.username}}</a></span>
-        </div>
+    <div *ngFor="let member of members">
+        <img [src]="member.profilePictureUrl"><span><a [href]="'/user/' + member.id">{{member.username}}</a></span>
     </div>
-    <ng-template #loading>
-        Loading ...
-    </ng-template>
 </div>

--- a/src/app/components/user-list/user-list.component.ts
+++ b/src/app/components/user-list/user-list.component.ts
@@ -8,7 +8,7 @@ import { UserAccount } from 'src/app/models/classes/UserAccount';
 })
 export class UserListComponent implements OnInit {
 
-  @Input() members$: UserAccount[] | Promise<UserAccount[]>;
+  @Input() members: UserAccount[];
 
   constructor() { }
 

